src/test/java/org/folio/utility/VertxUtility.java

```diff
@@ -0,0 +1,171 @@
+package org.folio.utility;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.support.HttpClient;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.ResponseHandler;
+import org.folio.rest.tools.utils.NetworkUtils;
+
+public class VertxUtility {
```

Comment:
This class seems to be more related to a RMB module rather than vert.x.

Please rename it to describe this scope more specifically e.g. ModuleUtility. Alternatively, split these methods into two classes, one for vert.x and another for the RMB module specific stuff e.g. enabling the module for a tenant